charts/kafka/chart/values.yaml

```
@@ -72,12 +72,32 @@ cp-helm-charts:
       url: http://kafka-cp-schema-registry:8081
     kafka:
       bootstrapServers: my-cluster-kafka-bootstrap:9092
+    affinity:
+      podAntiAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+                - key: app
+                  operator: In
+                  values:
+                    - cp-ksql-server
+            topologyKey: "kubernetes.io/hostname"
   cp-schema-registry:
     enabled: true
     image: confluentinc/cp-schema-registry
     imageTag: 6.2.1
     kafka:
       bootstrapServers: my-cluster-kafka-bootstrap:9092
+    affinity:
+      podAntiAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+                - key: app
+                  operator: In
+                  values:
+                    - cp-schema-registry
```

The `matchLabels` syntax is a bit easier to read IMO:
```suggestion
              matchLabels:
                app: cp-schema-registry
```